### PR TITLE
Login and Register Pages Completed (with Oauth2.0 google signin button)

### LINKED ISSUE #41: Allow user to register themselves as a small business owner or volunteer
**User story**
As a user, I want to be able to **sign up for a new account** and declare myself as a small business owner or volunteer so that I am provided with the right information and able to use the application as an authenticated user e.g. request help as a small business owner, mark myself available as a volunteer.

**Acceptance criteria**
- Ensure that user is redirected to `Login` (name tentative) page when they hit `/login`
- Able to provide essential information such as (but not limited to):
  - Name
  - Email adress
  - User name (or just take email address since they are guaranteed to be unique)
  - Password
  - Profile image (**absolutely a stretch goal**; do not worry too much about it unless it is straight-forward!)
- Ensure that when the user clicks `Create Account` (name tentative), a new user is created and stored in the redux layer

### PR DESCRIPTION
**Pull request by johnskimca**
#### [Title](#)
Login and Register Pages Completed (with Oauth2.0 google signin button)

**Checklist**
- [ ] Run `npm run lint:fix` or `yarn lint:fix`
**Done**

**What and how**
Created the login and sign up pages which includes the component Hyesun made. The pages are forms that have stated which will be connected by redux).
**Screenshots**

image
image

### CODE REVIEW

File: api/db/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const mongoose = require('mongoose')
 const path = require('path')
+// Committ
```
Comment (alice-0-kim) [inline api/db/index.js:3]: Bump

Comment (yehee) [inline api/db/index.js]: Bump!

File: client/src/components/Form/BusinessForm.js [hunk 2]
```diff
@@ -112,7 +112,7 @@ const Form = ({ open = false, closeForm, addBusiness }) => {
                     imgExtension={['.jpg', '.jpeg', '.gif', '.png', '.gif']}
                     maxFileSize={5242880}
                     singleImage
-                    withIcon
+                    withIconz
```
Comment (yehee) [inline client/src/components/Form/BusinessForm.js]: I think this is a typo?

File: client/src/components/LoginByGoogle.js [hunk 3]
```diff
@@ -35,15 +26,7 @@ const LoginByGoogle = () => {
...
-                            render={renderProps => (
-                                <button
-                                    type="button"
-                                    className="logout-button"
-                                    onClick={renderProps.onClick}
-                                >
-                                    Log Out
-                                </button>
-                            )}
+                            buttonText="Login"
```
Comment (yehee) [inline client/src/components/LoginByGoogle.js]: Should this be Logout?

**Comment by yehee** [comment]
LGTM just a minor thing then :shipit:

File: api/db/index.js (hunk 1 above)
Comment (johnskimca) [reply api/db/index.js]: Gone!

File: client/src/components/Form/BusinessForm.js (hunk 2 above)
Comment (johnskimca) [reply client/src/components/Form/BusinessForm.js]: Not sure how that happened but changed!

File: client/src/components/LoginByGoogle.js (hunk 3 above)
Comment (johnskimca) [reply client/src/components/LoginByGoogle.js]: Correct, changed!

**Comment by johnskimca** [comment]
Hey Alice, I cleared your review notes but as I pushed to my branch again I think it's asking for another review? Do you want me to just squash and merge as Admin or can you give your ok?

**Review by yehee — APPROVED** [review]

**Pull request merged**